javaparser-core/src/main/java/com/github/javaparser/ast/expr/StringLiteralExpr.java

```diff
@@ -78,6 +78,10 @@ public boolean remove(Node node) {
         return super.remove(node);
     }
 
+    public String asString() {
+        return value;
+    }
```

Comment:
Oh year forgot that, that would also be my question, does StringLiteral need such methods (well like you said yes). If `asString` should always escape, `StringLiteralExpr(String)` shouldn't. Maybe we add documentation to `getValue` that it always returns unescaped strings?